docs/docs/integrations/document-loaders/file-system.md

```diff
@@ -17,8 +17,8 @@ sidebar_position: 3
 
 ## APIs
 
-- `FileSystemDocumentLoader`
-
+- `ClassPathSource` / `ClassPathDocumentLoader`

```

Comment:
nit: we should probably add a separate doc file for this?

Reply:
We should also list it in https://docs.langchain4j.dev/tutorials/rag#document-loader

Reply:
Ahh you're probably right. I think I forgot this because my original implementation was just to re-use `FileSystemDocumentLoader`.

I'll go ahead and do this, as well as listing it in the tutorial.